scripts/exgdas_atmos_nceppost.sh

```diff
@@ -16,6 +16,8 @@ echo " Feb 18 - Meng - Removed legacy setting for generating grib1 data"
 echo "                 and reading sigio model outputs."
 echo " Aug 20 - Meng - Remove .ecf extentsion per EE2 review."
 echo " Sep 20 - Meng - Update clean up files per EE2 review."
+echo " Oct 21 - Meng - Remove jlogfile for wcoss2 transition."
+echo " Nov 21 - Meng - Update POSTGRB2TBL default setting for wcoss2 transition."
```

Comment:
@WenMeng-NOAA don't see this change in the script part, just want to double check with you

Reply:
@HuiyaChuang-NOAA Thanks for catching this. Actually the changes were already applied but the change logs were not updated. I tweaked them now.